### ISSUE TITLE
3.0 - Show tax amounts on orders & refunds if taxes are disabled, but the order did have taxes applied at an earlier date

### ISSUE THREAD

**Issue by ashleyfae** [easydigitaldownloads/easy-digital-downloads#7799]
- Enable taxes.
- Make a purchase that charges tax.
- Refund the order.
- You'll see tax amounts on both the order and refund.
- Disable taxes.
- Tax amounts disappear from the order and refund.

I think if the tax amount is > 0 then we should always show those records on the order, regardless of site settings.

If the tax amount is 0, then we conditionally check if taxes are enabled or not.

---

I don't think we necessarily have to change this for 3.0 itself because it's unlikely that a store owner will charge tax, then stop charging tax. But worth addressing later down the line, I think.

**Comment by spencerfinnell** [comment]
Good idea. In theory we can expand this conditional to check the current Order's tax amount: https://github.com/easydigitaldownloads/easy-digital-downloads/blob/release/3.0/includes/admin/payments/orders.php#L667

**Comment by robincornett** [comment]
Noting that this behavior exists in EDD 2.9.x as well.

**Comment by ashleyfae** [comment]
Thanks @robincornett . Going to punt this for now; we can address it shortly after 3.0.

**Comment by robincornett** [comment]
Noting that this is partially addressed in PR #8279, but only for the edit order screen, not the refund screen.
